src/Tomcat/TomcatController.ts

```diff
@@ -247,28 +259,14 @@ export class TomcatController {
     }
 
     private async startTomcat(serverInfo: TomcatServer, appName: string): Promise<void> {
-        let statusBar: vscode.StatusBarItem;
```

Comment:
As we delete these lines, where does the logic go?

Reply:
status bar command was used to open war package url in browser in previous behavior. Now we are supporting showing war packages directly in sub-tree and a context command to "open in browser".